.gitignore

```diff
@@ -20,4 +20,6 @@ cabal.project.local
 cabal.project.local~
 .HTF/
 .ghc.environment.*
-.cache
\ No newline at end of file
+.cache
+.history
```

Comment:
`.history` comes from the repl and `stack.yaml.lock` comes from newer stack versions, I guess.